.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - master
-  pull_request:
+  pull_request_target:
```

Comment:
This is a cool feature. However, I don't think simply switching to this is sufficient, as it will mean that we will be testing the code on the target branch and not the PR branch. I think we need to change all the checkout actions:
```yaml
uses: actions/checkout@v2
with:
  ref: ${{github.event.pull_request.head.ref}}
  repository: ${{github.event.pull_request.head.repo.full_name}}
```
However, having to change all the jobs is messy. I think we can use this new `pull_request_target` feature to change the existing `cleanup` workflow to use that trigger (rather than a schedule), then we don't need to change anything in this job.